MovieQuizTests/MovieQuizPresenterTests.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  MovieQuizViewControllerTests.swift
+//  MovieQuizTests
+//
+//  Created by Юрченко Артем on 07.11.2022.
+//
+
+import XCTest
+@testable import MovieQuiz
+
+final class MovieQuizViewControllerProtocolMock: MovieQuizViewControllerProtocol {
+    func show(quiz step: QuizStepViewModel) {
+        
+    }
```

Comment:
Еще один вариант перечисления функций-заглушек, более компактный для чтения
```suggestion
    func show(quiz step: QuizStepViewModel) { }
```